server/command/command.go

```diff
@@ -171,14 +84,18 @@ func (c *Command) validate() (string, []string, error) {
 	return command, split[1:], nil
 }
 
-func (c *Command) handleCommand(
+func (c *Command) run(
 	subcommands map[string]func([]string) (md.MD, error),
 	parameters []string,
 ) (md.MD, error) {
 	if len(parameters) == 0 {
 		return c.subUsage(subcommands), errors.New("expected a (sub-)command")
 	}
 
+	if parameters[0] == "help" {
```

Comment:
The usage still reports to use `--help`: https://github.com/levb/mattermost-plugin-solar-lottery/blob/lev-autopilot-1/server/command/command_flags.go#L36